env-react.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
     ],
     'parserOptions': {
         'ecmaFeatures': {
-            'jsx': true
+            'jsx': true,
+            'experimentalObjectRestSpread': true
```

Comment:
this feature was added to eslint for react compat (really only reason people used `babel-eslint`), and is very prevalent in react code bases